### ISSUE TITLE
error while loading shared libraries

### ISSUE THREAD

**Issue by albertosantini** [electron-userland/electron-prebuilt#131]
On Windows 10 64bit with electron-prebuilt 1.1.0, I am facing with the following error:
```
λ node_modules/.bin/electron
C:/My/Programs/nodejs/node.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory
```

No problems during the installation.
With 1.0.2, the app works nice.

**Comment by bontibon** [comment]
I believe the issue I have might be related:

```
Error: A dynamic link library (DLL) initialization routine failed.
\\?\C:\Users\me\Desktop\app\node_modules\ref\build\Release\binding.node
    at Error (native)
    at process.module.(anonymous function) [as dlopen] (ELECTRON_ASAR.js:158:20)
    at Object.Module._extensions..node (module.js:440:18)
    at Object.module.(anonymous function) [as .node] (ELECTRON_ASAR.js:158:20)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at bindings (C:\Users\me\Desktop\app\node_modules\bindings\bindings.js:76:44)
    at Object.<anonymous> (C:\Users\me\Desktop\app\node_modules\ref\lib\ref.js:5:47)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19), message=A dynamic link library (DLL) initialization routine failed.
```
Windows 10 x64
electron-prebuilt: 1.1.0
Node 32-bit: v6.2.0

**Comment by albertosantini** [comment]
Fixed with 1.1.1.

Thanks for your time.
